dmci/distributors/pycsw_dist.py

```diff
@@ -0,0 +1,188 @@
+import logging
+import requests
+import xml
+
+from py_mmd_tools.xml_utils import xml_translate
+
+from dmci.distributors.distributor import Distributor, DistCmd
+
+logger = logging.getLogger(__name__)
+
+class PyCSWDist(Distributor):
+
+    TOTAL_DELETED = 'total_deleted'
+    TOTAL_INSERTED = 'total_inserted'
+    TOTAL_UPDATED = 'total_updated'
+    STATUS = [TOTAL_DELETED, TOTAL_INSERTED, TOTAL_UPDATED]
+
+    def __init__(self, cmd, xml_file=None, metadata_id=None, **kwargs):
+
+        super().__init__(cmd, xml_file, metadata_id, **kwargs)
+
+    def run(self):
```

Comment:
docstring missing

Reply:
solved in 3da4ca046dd032a97e2df21b42d8b6fe1e77cf5a